inject/inject.vala

```diff
@@ -20,6 +21,21 @@ namespace Frida.Inject {
 		{ null }
 	};
 
+	private static string read_stdin () {
```

Comment:
Should be moved below `main()`, so the code can be read like a newspaper – top to bottom.